### PR TITLE
Changelog generation: remove “experimental” status from WP 5.8 stable items

### PR DESCRIPTION
**Pull request by getdave**
## Description
Working on Gutenberg release `11.0.0` I kept having to move items out of experimental as they were now considered stable. This included:

* Widgets
* Query block
* Some `post-*` blocks.

This PR updates the script that auto generates the changelog to avoid grouping those items in the "Experimental" section. 

## How has this been tested?

Not actually sure how best to test. @ockham any ideas here?

## Types of changes
 Bug fix (non-breaking change which fixes an issue)

## Checklist:
- [ ] My code is tested.
- [ ] My code follows the WordPress code style. 
- [ ] My code follows the accessibility standards. 
- [ ] I've tested my changes with keyboard and screen readers. 
- [ ] My code has proper inline documentation. 
- [ ] I've included developer documentation if appropriate. 
- [ ] I've updated all React Native files affected by any refactorings/renamings in this PR (please manually search all `*.native.js` files for terms that need renaming or removal).

### CODE REVIEW

File: bin/plugin/commands/changelog.js [hunk 1]
```diff
@@ -61,18 +61,15 @@ const manifest = require( '../../../package.json' );
...
 	'[Block] Post Hierarchical Terms': 'Experiments',
 	'[Block] Post Title': 'Experiments',
-	'[Block] Site Logo': 'Experiments',
 	'[Feature] Full Site Editing': 'Experiments',
 	'Global Styles': 'Experiments',
 	'[Feature] Navigation Screen': 'Experiments',
-	'[Feature] Widgets Screen': 'Experiments',
 	'[Package] Dependency Extraction Webpack Plugin': 'Tools',
 	'[Package] Jest Puppeteer aXe': 'Tools',
 	'[Package] E2E Tests': 'Tools',
```
Comment (getdave) [inline bin/plugin/commands/changelog.js]: @priethor Anything else you're aware of we should remove the `Experiments` label from?

Comment (priethor) [reply bin/plugin/commands/changelog.js]: `Post Featured` Image and `Post Title` are also making it in 5.8 and should not be experimental anymore.

With the Global Styles framework merged, would it make sense to mark it as non-experimental, even without the UI?

**Review by priethor — CHANGES_REQUESTED** [review]

**Comment by getdave** [comment]
> With the Global Styles framework merged, would it make sense to mark it as non-experimental, even without the UI?

@priethor I was thinking perhaps as the UI isn't finalised it's ok to keep Experimental and leave the release lead to move based on human understanding of whether it's a framework or a UI part.

> Post Featured Image and Post Title are also making it in 5.8 and should not be experimental anymore.

Updated ✅

**Review by priethor — APPROVED** [review]
LGTM! Thanks!

**Comment by getdave** [comment]
Rebase to try and force the tests to pass...

**Comment by ockham** [comment]
> ## How has this been tested?
>
> Not actually sure how best to test. @ockham any ideas here?

You've probably found it by now, but you can run

```
npm run changelog -- --milestone="Gutenberg 11.0" --unreleased
```

It might be a bit tricky to run a version of that command that will include PRs with one of the labels that this PR moves out of Experimental. You could try removing the `--unreleased` flag to include the PRs that went into the RC:

```
npm run changelog -- --milestone="Gutenberg 11.0"
```

One of those hopefully touched one of the relevant blocks (e.g. Site Logo) 😄

**Comment by getdave** [comment]
I tested this using @ockham's suggestion and none of the items I've removed from the Experiments list in this PR showed up in the generated changelog under `Experiments` for `11.0.0`.

**Pull request merged**